pkgs/by-name/te/testkube/package.nix

```
@@ -0,0 +1,36 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+buildGoModule rec {
+  pname = "testkube";
+  version = "2.1.56";
+
+  src = fetchFromGitHub {
+    owner = "kubeshop";
+    repo = "testkube";
+    rev = "v${version}";
+    sha256 = "sha256-P+A9lUMzQ3M0SEVZBMDSMj8S0uCsEhadv5vDRxbQORA=";
+  };
+
+  vendorHash = "sha256-44aIwddMH6CMfTno90xGkHgna4DO2Ii3KhpMwv6Zjmo=";
+
+  ldflags = [
+    "-X main.version=${version}"
+    "-X main.builtBy=nixpkgs"
+    "-X main.commit=v${version}"
+    "-X main.date=1970-01-01-00:00:01"
+  ];
+
+  doCheck = false;
```

elaborate why you disabled the test in comment, for example (just imaginary)

```suggestion
  doCheck = false; # Require binding to kubernets service port
```

---

Fixed in feb8a2753c1f2c9fd5a35edc9b4fd26c293ac1e4. Don't remember why I skipped checks in my personal configs. Build works with check phase now.